tasks/client_keys.yml

```diff
@@ -0,0 +1,87 @@
+# generate client keys
+---
+- name: Copy openssl client extensions
+  copy:
+    src: openssl-client.ext
+    dest: "{{ openvpn_key_dir }}"
+    owner: root
+    group: root
+    mode: 0400
+
+- name: Generate client key
+  command: "openssl req -nodes -newkey rsa:{{ openvpn_rsa_bits }} -keyout {{ item }}.key -out {{ item }}.csr -days 3650 -subj /CN=OpenVPN-Client-{{  inventory_hostname[:24]  }}-{{  item[:24]  }}/"
+  args:
+    chdir: "{{ openvpn_key_dir }}"
+    creates: "{{ item }}.key"
+  with_items: "{{ openvpn_clients }}"
+
+- name: Protect client keys
+  file:
+    path: "{{ openvpn_key_dir }}/{{ item }}.key"
+    mode: 0400
```

Comment:
You could also force user and group here

Reply:
fixed